atd-vze/src/views/Crashes/Crash.js

```diff
@@ -254,14 +254,17 @@ function Crash(props) {
                 </Col>
               </Row>
             </CardHeader>
-            <CardBody style={{ minHeight: "350px" }}>
-              {(!latitude || !longitude) && (
+            <CardBody
+              className="d-flex flex-column"
+              style={{ minHeight: "400px" }}
```

Comment:
This fixes a bug where the map was escaping its parent card when the "no coordinates" alert showed. There is a screenshot in the issue of this issue. Now, the map will stretch to fit the remaining height in the card. The min height is still needed so the map doesn't shrink down when there is no crash diagram available to stretch the parent row to have height.

Makes me wonder what making the height of this row fixed might look like so it doesn't look different on every crash? There might be a good reason to keep it this way, though.

Makes me wonder what a fixed height for both of these could look like!

Reply:
I think that's a great idea. There is some weird re-sizing that happens on save as well. I think we can revisit this in the near-ish future as we take a closer look at this whole page's layout 🙏